news-search/src/pages/SharedLayout.js

```diff
@@ -2,10 +2,16 @@ import { Outlet} from 'react-router-dom'
 import NavbarExpand from '../components/Navbar';
 import Footer from '../components/Footer';
 
-const SharedLayout = () => {
+const SharedLayout = ({changeTopicBitcoin, changeTopicAbortion, changeTopicInflation,changeTopicGlobalWarming, allTopics}) => {
```

Comment:
again a shared topic object might be future proof for adding new topics later on